test/e2e/test_job_submissions.py

```diff
@@ -406,6 +406,9 @@ def sessions_exist(current_job: Job) -> bool:
 
         LOG.info(f"Job result: {job}")
 
+        # Wait until the envExit runs as well
+        time.sleep(10)
```

Comment:
Is anything actually happening on exit that we need to sleep this long? I see that it waits for the job to complete above. Should that function not already make sure that the job has finished it's calls before returning?

Reply:
The job is considered completed even without running the envExit and envExit is not required to finish for the job to finish.